### PR TITLE
Add Tour of LC and TP content for TESS

### PR DESCRIPTION
**Pull request by mustaric**
Here is a TESS notebook to go through the LC and TP files. It has not been run and the files point to a public set of data files on line. So hopefully it works. 
Repository includes the notebook, one image for the footer and a requirements.txt file.

### CODE REVIEW

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 1]
```diff
@@ -0,0 +1,922 @@
...
+    "sapflux=lcdata['SAP_FLUX']\n",
+    "pdcflux=lcdata['PDCSAP_FLUX']\n",
+    "quality=lcdata['QUALITY']\n",
+    "time=lcdata['TIME']\n",
+    "\n",
+    "fig=plt.figure(figsize=(11,4))\n",
+    "fig.add_subplot(211)\n",
+    "plt.plot(time,sapflux,'.',label='SAP')\n",
+    "plt.legend(loc='lower left')\n",
+    "#Zoom in to see anything.\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before the comment for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 2]
```diff
@@ -0,0 +1,922 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Same plot as before with flagged data removed.\n",
+    "fig=plt.figure(figsize=(14,5))\n",
+    "fig.add_subplot(211)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 3]
```diff
@@ -0,0 +1,922 @@
...
+    "plt.plot(time[~bad_data],sapflux[~bad_data],'.',label='SAP')\n",
+    "plt.legend(loc='lower left')\n",
+    "fig.add_subplot(212)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 4]
```diff
@@ -0,0 +1,922 @@
...
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# sapflux is the photometry calculated by the pipeline.\n",
+    "# opap_flux is the photometry we calculated from the pixels.\n",
+    "fig=plt.figure(figsize=(11,5))\n",
+    "fig.add_subplot(211)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 5]
```diff
@@ -0,0 +1,922 @@
...
+    "plt.plot(time[~bad_data],sapflux[~bad_data],'r.',label=\"SAP_FLUX\")\n",
+    "plt.plot(time[~bad_data],opap_flux[~bad_data],'b.',ms=2,label=\"Target Pixel Op. Aperture\")\n",
+    "plt.xlabel(\"Time (BTJD)\")\n",
+    "plt.ylabel(\"Flux (e-/s)\")\n",
+    "plt.legend(loc=\"lower left\")\n",
+    "plt.xlim(1326,1330)\n",
+    "plt.ylim(7955,8090)\n",
+    "fig.add_subplot(212)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 6]
```diff
@@ -0,0 +1,922 @@
...
+   "outputs": [],
+   "source": [
+    "#Solution to (a)\n",
+    "#Here we choose do a simple normalization by subtracting off the median.\n",
+    "normflux=pdcflux[~bad_data]-np.nanmedian(pdcflux[~bad_data])\n",
+    "fwNorm_row=fluxcent_row[~bad_data]-np.nanmedian(fluxcent_row[~bad_data])\n",
+    "fwNorm_col=fluxcent_col[~bad_data]-np.nanmedian(fluxcent_col[~bad_data])\n",
+    "\n",
+    "plt.figure(figsize=(10,5))\n",
+    "plt.subplot(121)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb [hunk 7]
```diff
@@ -0,0 +1,922 @@
...
+    "plt.plot(fwNorm_row,normflux,'b+')\n",
+    "plt.title(\"row\")\n",
+    "plt.xlabel(\"change in row(pixels)\")\n",
+    "plt.ylabel(\"change in flux\")\n",
+    "plt.subplot(122)\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/beginner_tour_lc_tp.ipynb]: Suggest newline before this line for readability.

**Comment by mustaric** [comment]
I think I made all your grammatical and styling changes. Let me know how it looks now.

File: notebooks/MAST/TESS/beginner_tour_lc_tp/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,3 @@
+astropy==3.1.dev22604
+matplotlib==2.2.2
+numpy==1.15.1
```
Comment (ceb8) [inline notebooks/MAST/TESS/beginner_tour_lc_tp/requirements.txt]: Sorry I didn't catch this earlier Susan. (This fix does not involve touching your notebook though).

These should just be the lowest "known good" version and should not include dev versions. So if you could change the "==" to ">=" and the astropy version to 3.0.4 (I tested on that version so I know it works), that would be awesome.

**Comment by ceb8** [comment]
@mustaric Looks great!  I just had one more comment on your requirements.txt file after seeing @eteq's comment on @scfleming's last PR.

**Comment by ceb8** [comment]
@mustaric Thanks!  Yeah, I suggested that particular version because I ran your notebook with it. Merging now.

**Pull request merged**